changelog/256.feature.rst

```diff
@@ -0,0 +1 @@
+Add a ``-q`` option to progressively silence tox's output. By @sigmavirus24
```

Comment:
I feel like we needs some better description here, what does progressively  exactly means? would prefer to detail what's the interaction between ``-v`` and ``-q``; and for what amount what output the users is expected to get